source/inventory_api.rst

```diff
@@ -0,0 +1,461 @@
+************************************
+Inventory API
+************************************
+
+
+`1. Add / Update Inventory`_
+
+`2. Update Out of Stock Inventory`_
+
+
+1. Add / Update Inventory
+==================
+| URL endpoint: https://host.com/api/backend/inventories/update
+| Allowed method: POST
+| Requires authentication: Yes
+
+A. Request
+----------
+
+You can add or update a product variant in the store inventory by calling the API with these parameters
```

Comment:
`add or update an inventory for each variant`, it's add / update information of the `inventory`, not the `product` or `variant`

Reply:
The above means adding a/multiple variants into the store inventory, or update already existing variants in the store inventory

after testing the api, if I give an empty array, it does not remove any variants, this only seems to add variants if they dont exist in the inventory, or update existing variant's fields.

Reply:
unless it is meant that 1 "inventory" is a variant

Reply:
An inventory is by definition an array of goods, not one good.
Inventories (plural) describes multiple arrays of goods 
example:
( this store has an inventory of variants), these store's inventories are stocked up, one variant of this store's inventory is out of stock. )

Reply:
the "information" of the inventory, are the variants that exist within it.

Reply:
I have updated it to make my statement more clear.

Reply:
@vincentiusgunawan is this correct (check my updated statement)

Reply:
For semantics I would have renamed "inventories" into inventory

Reply:
<img width="1012" alt="Screen Shot 2019-08-01 at 3 32 59 PM" src="https://user-images.githubusercontent.com/52644069/62278159-bc99a200-b471-11e9-8cd6-61b803b0e9ed.png">
should be renamed to inventory, and 
<img width="942" alt="Screen Shot 2019-08-01 at 3 33 10 PM" src="https://user-images.githubusercontent.com/52644069/62278177-c4594680-b471-11e9-944c-a8f324a1018d.png">
here I updated a variant (burger-1)'s price from 36364 to 36364444

Reply:
actually, you are right.
Inventory is part of the variant's information.
Okay, it's all good with this explanation.